src/constants.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+
+// const cst = () => {
+export let tempIncreaseButton = null;
+export let tempDecreaseButton = null;
+export let getRealTimeTemp = null;
+export let tempValue = null;
+export let gardenContent = null;
+export let temperature = 79;
+export let headerCity = null;
+export let cityNameUserInput = null;
+export let defaultCityName = 'Princeton';
+export let skyTemplate = null;
+export let skyOption = null;
+export let skyInput = null;
+export let resetCityName = null;
+export let realTimeTemperature = null;
+//     return tempDecreaseButton, tempIncreaseButton, getRealTimeTemp, tempValue, gardenContent, temperature, headerCity, cityNameUserInput, defaultCityName, skyTemplate, skyOption, skyInput, resetCityName, realTimeTemperature;
```

Comment:
I like the idea to use a constants file for the things like the sky settings that won't change, but it looks like the values in this file aren't being used. Were you running into issues reading from this file?